go.mod

```diff
@@ -48,7 +48,7 @@ require (
 	github.com/satori/go.uuid v1.2.0
 	github.com/scylladb/go-reflectx v1.0.1
 	github.com/shopspring/decimal v1.3.1
-	github.com/smartcontractkit/libocr v0.0.0-20210826183649-d12971936c12
+	github.com/smartcontractkit/libocr v0.0.0-20211117215336-6c9726817b2d
```

Comment:
TODO: Update to libocr master once release is out

Reply:
Is this a blocker on merge?

Reply:
yes